keras_cv/layers/preprocessing/gaussian_blur.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+
+@tf.keras.utils.register_keras_serializable(package="keras_cv")
+class GaussianBlur(tf.keras.__internal__.layers.BaseImageAugmentationLayer):
+    """Applies a Gaussian Blur with random sigma to an image.
+
+    Args:
+        kernel_size: int, 2 element tuple or 2 element list. x and y dimensions for
```

Comment:
Check out how we are handling sampling like this.  `sigma` could use this.  There is a nice utility to parse these ranges for you.  See https://github.com/keras-team/keras-cv/pull/238

This isn't merged yet - but you can rebase on the branch `factor` to use this.